app/models/type.rb

```diff
@@ -50,27 +52,19 @@ def copy_from_type(source_type)
                           join_table: "#{table_name_prefix}custom_fields_types#{table_name_suffix}",
                           association_foreign_key: "custom_field_id"
 
-  belongs_to :color,
-             optional: true,
-             class_name: "Color"
+  belongs_to :color, optional: true, class_name: "Color"
 
   acts_as_list
 
-  validates :name,
-            presence: true,
-            uniqueness: { case_sensitive: false },
-            length: { maximum: 255 }
+  validates :name, presence: true, uniqueness: { case_sensitive: false }, length: { maximum: 255 }
```

Comment:
❓ seeing this, does it mean, we have no work package type contract? We have to do the validations on the model creation?

Reply:
Ah, yeah, I checked the class, and I'd vote to move validation in there. Your call, if you want to do this with the next step (of pattern string validation itself).

Reply:
Oh, this validation was already there, I just re-formatted the line. XD

Reply:
true, then it is even less a reason to do this in this PR, but maybe worth a though, when adding pattern validation.